bridge-fungible-token/tests/harness.rs

```diff
@@ -879,18 +1057,26 @@ mod revert {
     #[should_panic(expected = "Revert(18446744073709486080)")]
     async fn verification_fails_with_wrong_sender() {
         let mut wallet = env::setup_wallet();
+        let configurables: Option<BridgeFungibleTokenContractConfigurables> = None;
```

Comment:
huh, this test is strange. How come it tests for the `18446744073709486080` revert rather than some custom error?

Reply:
Good catch. This could be updated. Will do that.